workflows/tools/validate_array_elements_coordinates.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.2
+class: CommandLineTool
+
+label: validate_array_elements_coordinates
+doc: |-
+    Validate array element coordinates by plotting.
+    Plot all array element coordinates in different coordinate systems
+    and with different zooming levels including the telescope altitudes.
+    Plot distance distributions between telescopes.
+
+baseCommand:
+  - python
+  - /workdir/gammasim-tools/applications/print_array_elements.py
+
+inputs:
+
+    - id: data
+      doc: |-
+        List of array elements in simulation system coordinates.
+      type: File
+
+arguments: ["--h"]
```

Comment:
Shouldn't this argument option be in the `baseCommand`?

Reply:
Yes. I guess what confused me is that when you use `echo` as a place holder it is under `baseCommand` rather than under `inputs`.

Reply:
Note that `arguments` is not under `inputs`, but a separate keyword.

Reply:
oh right, apologies for not noticing.